jmdaemon/jmdaemon/onionmc.py

```diff
@@ -154,6 +154,13 @@ def from_string_decode(cls, msg: bytes) -> 'OnionCustomMessage':
         return cls(text, msgtype)
 
 class OnionLineProtocol(basic.LineReceiver):
+    # there are messages requiring more than LineReceiver's 16KB,
+    # specifically, large coinjoin transaction `pushtx` messages.
+    # 50K is finger in the air for: 500bytes per participant, 40

```

Comment:
```suggestion
    # 40K is finger in the air for: 500bytes per participant, 40
```
Or am I missing something?

Reply:
Thanks fixed in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1262/commits/ff60a15db46383d070e95c870b614f21b7539edc